# ParticleArray ROS Message - represents an array of weighted poses
std_msgs/Header header
Particle[] particles
